pkg/volume/csi/csi_plugin.go

```diff
@@ -168,7 +167,6 @@ func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string,
 		klog.Error(log("registrationHandler.RegisterPlugin failed at CSI.NodeGetInfo: %v", err))
 		if unregErr := unregisterDriver(pluginName); unregErr != nil {
 			klog.Error(log("registrationHandler.RegisterPlugin failed to unregister plugin due to previous: %v", unregErr))
-			return unregErr
```

Comment:
If the CRD is not present, return the registration error and log the unregistration one (if any).